db/tools/bulk_patient_import.rb

```diff
@@ -0,0 +1,214 @@
+DB = 'localhost'
+USER = 'sa'
+PASSWORD = 'fake.fake.fake.1234'
```

Comment:
question:   is is possible to make these command line arguments?  I know nothing about ruby

Reply:
Added those changes and updated the invocations in the pr comments